pkg/appgw/frontend_listeners_test.go

```diff
@@ -444,4 +444,72 @@ var _ = Describe("MutateAppGateway ingress rules and parse frontend listener con
 			Expect(*listeners).To(ContainElement(expectedListener443))
 		})
 	})
+
```

Comment:
we should also add a positive test ?

Reply:
> we should also add a positive test ?

The positive case is already covered in functional test regarding waf policy